cve_bin_tool/theme.py

```diff
@@ -0,0 +1,12 @@
+from rich.theme import Theme
+
+# Rich theme to colorize in the terminal
+cve_theme = Theme(
+    {
+        "critical": "red",
+        "high": "blue",
+        "medium": "yellow",
+        "low": "green",
+        "unknown": "white",
+    }
+)
```

Comment:
Thanks to this, I guess now We'll be able to provide the user with an option to Customise the console colours. Yay!